cmake/Corrosion.cmake

```diff
@@ -1290,6 +1299,20 @@ function(corrosion_install)
                             DESTINATION ${DESTINATION}
                             ${CONFIGURATIONS}
                     )
+
+                    if(EXPORT_NAME)
+                        get_target_property(COR_FILE_NAME ${INSTALL_TARGET}-static COR_FILE_NAME)
+                        file(APPEND
+                            ${CMAKE_BINARY_DIR}/corrosion/${EXTRA_TARGETS_EXPORT_NAME}
+"
+add_library(${INSTALL_TARGET}-static STATIC IMPORTED)
+set_target_properties(${INSTALL_TARGET}-static
+    PROPERTIES
+    IMPORTED_LOCATION \${PACKAGE_PREFIX_DIR}/${DESTINATION}/${COR_FILE_NAME}
```

Comment:
Anything that can be a filepath needs to be quoted, since filepaths can contain spaces.